aries_cloudagent/wallet/routes.py

```diff
@@ -355,13 +355,20 @@ async def wallet_create_did(request: web.BaseRequest):
                 f" support key type {key_type.key_type}"
             )
         )
+    seed = None
+    if context.settings.get("wallet.allow_insecure_seed"):
+        seed = body.get("seed") or None
```

Comment:
Would be good to throw an error if a seed is provided and `wallet.allow_insecure_seed` is not enable to make it more clear to the end user the seed is not used. This can give the false sense the seed is being used to create the did.

Reply:
@TimoGlastra can you suggest the error message to be displayed for this. Can it be a simple HTTPBadRequest: Seed not supported ?